modules/javafx.graphics/src/main/java/com/sun/javafx/font/coretext/CTFactory.java

```diff
@@ -50,6 +50,15 @@ private CTFactory() {
      */
     CTFontFile createFontFile(String name, long fontRef) throws Exception {
         String filename = OS.CTFontCopyURLAttribute(fontRef);
+        // macOS 13 Ventura reports ".ThonburiUI Regular Regular" as the name
+        // of font used for Thai. Per inspection of the font this is incorrect.
+        // Need to do fix up here. I 
+        if (name.endsWith(" Regular Regular")) {
+            System.err.println("Fix up double use of Regular in name : " + name);
+            if (debugFonts) {
+            }
```

Comment:
I guess you meant to put the print inside the `if (debugFonts)`?

Reply:
fixed